security-oauth2/src/main/java/io/micronaut/security/oauth2/endpoint/token/request/context/OpenIdCodeTokenRequestContext.java

```diff
@@ -56,19 +53,6 @@ public OpenIdCodeTokenRequestContext(AuthorizationResponse authorizationResponse
         this.oauthRouteUrlBuilder = oauthRouteUrlBuilder;
     }
 
-    /**
-     * Resolves the media type for the request body
-     *
-     * @param clientConfiguration The client configuration
-     * @return The media type
-     */
-    protected static MediaType getMediaType(OauthClientConfiguration clientConfiguration) {
```

Comment:
Moving this to `AbstractTokenRequestContext` doesn't make sense because this logic is specific to openid

Reply:
Fixed in new commit.